Apps/WebClient/src/ClientApp/package.json

```diff
@@ -62,9 +62,9 @@
     },
     "dependencies": {
         "@chenfengyuan/vue-countdown": "^1.1.5",
-        "@fortawesome/fontawesome-svg-core": "1.2.36",
-        "@fortawesome/free-regular-svg-icons": "^5.15.4",
-        "@fortawesome/free-solid-svg-icons": "^5.15.4",
+        "@fortawesome/fontawesome-svg-core": "1.3.0",
```

Comment:
This ticket looks like it included both Admin and WebClient.

Reply:
Changes applied to Admin.